examples/multi_python_versions/MODULE.bazel

```diff
@@ -56,6 +57,11 @@ pip.parse(
     python_version = "3.11",
     requirements_lock = "//requirements:requirements_lock_3_11.txt",
 )
+pip.parse(
+    hub_name = "pypi_3_10_only",
+    python_version = "3.11",
```

Comment:
This should be `3.10` and not `3.11`.

Reply:
ah, good catch. Fixed this up but I get the same error with 3.11 showing 3.10 now.